modules/developers-guide/pages/tutorials/scalability-cancan.adoc

```diff
@@ -0,0 +1,103 @@
+= Creating scalable apps
+ifdef::env-github,env-browser[:outfilesuffix:.adoc]
+:toc:
+:toc: right
+:toc-title: TUTORIAL - SCALABILITY
+:toclevels: 3
+:proglang: Motoko
+:platform: Internet Computer platform
+:IC: Internet Computer
+:company-id: DFINITY
+:sdk-short-name: DFINITY Canister SDK
+:sdk-long-name: DFINITY Canister Software Development Kit (SDK)
+
+The CanCan sample application is a simplified video-sharing service that demonstrates several features that you can use as models for your own applications.
+For example, here are a few things you can learn by exploring the CanCan sample application:
+
+* How to build a scalable application by splitting content into fragments for upload and storage then using queries to retrieve and reassemble the fragments for efficient streaming.
+
+* How to configure interoperability for an application that uses canisters written in different back-end languages.
+
+* How to implement a basic access control model for authenticating which videos are owned by which end-users.
+
+* How to build a front-end that implements more sophisticated user interface features for desktop or mobile apps.
+
+== Splitting uploaded content into multiple canisters
+
+Because canisters are compiled WebAssembly modules, they have certain known limitations. 
+For example, Web Assembly modules have a maximum of 4GB for memory and an upper limit on the number of object calls allowed.
```

Comment:
I would note that these are *current* limitations of WebAssembly.

Reply:
Done